example.json

```diff
@@ -20,7 +20,7 @@
     ],
     "anEmptyValue": "",
     "sops": {
-        "version": 1.6,
+        "version": "1.6",
```

Comment:
I've changed the example to use a string version field, because otherwise it doesn't work with new SOPS versions